test/integration/features/cmd-image.feature

```diff
@@ -58,7 +58,9 @@ Feature: Basic image caching test
       And JSON config file "config/config.json" contains key "cache-images" with value matching "[alpine:latest]"
 
      When executing "minishift start" succeeds
-     Then stdout of command "minishift image list --vm" contains "alpine:latest"
+     Then Minishift should have state "Running"
+      And image export completes with 4 images
```

Comment:
Just realized this might be problematic, if we run this without the `basic.feature` it will fail. In test/integration/integration_test.go in cleanTestDirConfiguration() we should probably add `cache/images` to be also removed after the feature to secure isolation between features.